Eric Vyncke: CID field optional (#494)
quicwg/ops-drafts: draft-ietf-quic-manageability.md

  (see {{Section 22.2 of QUIC-TRANSPORT}}).

- source and destination connection ID: short and long headers carry a

>   destination connection ID, a variable-length field which can also be zero-length
>   but if present can be used to

Reviewer: ```suggestion
  destination connection ID, a variable-length field which, if of non-zero
  length, can be used to
```

Reviewer: (not sure this is better)

Author: Maybe this?
```suggestion
  destination connection ID, a variable-length field which, if not zero-length,
  can be used to
```
